test/infrastructure/inmemory/api/v1alpha1/inmemorymachine_types.go

```diff
@@ -28,11 +28,113 @@ const (
 	MachineFinalizer = "inmemorymachine.infrastructure.cluster.x-k8s.io"
 )
 
+const (
+	// VMProvisionedCondition documents the status of the provisioning VM implementing to the InMemoryMachine.
+	VMProvisionedCondition clusterv1.ConditionType = "VMProvisioned"
+
+	// WaitingForClusterInfrastructureReason (Severity=Info) documents an InMemoryMachine VM waiting for the cluster
+	// infrastructure to be ready.
+	WaitingForClusterInfrastructureReason = "WaitingForClusterInfrastructure"
+
+	// WaitingControlPlaneInitializedReason (Severity=Info) documents an InMemoryMachine VM waiting
+	// for the control plane to be initialized.
+	WaitingControlPlaneInitializedReason = "WaitingControlPlaneInitialized"
+
+	// WaitingForBootstrapDataReason (Severity=Info) documents an InMemoryMachine VM waiting for the bootstrap
+	// data to be ready before starting to create the CloudMachine/VM.
+	WaitingForBootstrapDataReason = "WaitingForBootstrapData"
+
+	// VMProvisioningReason (Severity=Info) documents a InMemoryMachine VM provisioning.
+	VMProvisioningReason = "WaitingForStartupTimeout"
```

Comment:
Usually the constant names match the string, this is not the case here. Which one do we want to use?

Same for the other Provisioning reasons below

Reply:
fixed